assets/src/components/content/add_resource_content/AddActivity.tsx

```diff
@@ -17,15 +17,17 @@ export const AddActivity: React.FC<Props> = ({ resourceContext, onAddItem, edito
       const enabled = editorDesc.globallyAvailable || editorDesc.enabledForProject;
 
       return enabled ? (
-        <a
-          href="#"
+        <button
```

Comment:
Changing links to buttons fixed the "jumps to top of page when adding content" issue